experiments/torchgeo/run_l7irish_experiments.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""Runs the train script with a grid of hyperparameters."""
+import itertools
+import os
+import subprocess
+from multiprocessing import Process, Queue
+
+# list of GPU IDs that we want to use, one job will be started for every ID in the list
+GPUS = [0]
+DRY_RUN = False  # if False then print out the commands to be run, if True then run
+DATA_DIR = "/scratch.local/yichia3/l7irish_cog_3857"  # path to the L7Irish data directory
+
+# Hyperparameter options
+# model_options = ["resnet18", "resnet50"]
+model_options = ["unet"]
+backbone_options = ["resnet50"]
+lr_options = [1e-4]
+loss_options = ["ce"]
+weight_options = ["imagenet"]
+# weight_options = ["imagenet_only", "random", "imagenet_and_random", "random_rgb"]
+
+
+def do_work(work: "Queue[str]", gpu_idx: int) -> bool:
+    """Process for each ID in GPUS."""
+    while not work.empty():
+        experiment = work.get()
+        experiment = experiment.replace("GPU", str(gpu_idx))
+        print(experiment)
+        if not DRY_RUN:
+            subprocess.call(experiment.split(" "))
+    return True
+
+
+if __name__ == "__main__":
+    work: "Queue[str]" = Queue()
+
+    for model, backbone, lr, loss, weights in itertools.product(
+        model_options, backbone_options, lr_options, loss_options, weight_options
+    ):
+        experiment_name = f"{model}_{backbone}_{lr}_{loss}_{weights.replace('_','-')}"
+
+        output_dir = os.path.join("output", "l7irish_experiments")
+        log_dir = os.path.join(output_dir, "logs")
+        config_file = os.path.join("conf", "l7irish.yaml")
+
+        if not os.path.exists(os.path.join(output_dir, experiment_name)):
+            command = (
+                "python train.py"
+                + f" config_file={config_file}"
+                + f" experiment.name={experiment_name}"
+                + f" experiment.module.segmentation_model={model}"
+                + f" experiment.module.learning_rate={lr}"
+                + f" experiment.module.loss={loss}"
+                + f" experiment.module.weights={weights}"
+                + f" experiment.module.backbone={backbone}"
+                + f" experiment.datamodule.weights={weights}"
```

Comment:
There is no experiment level anymore, these should just be module and datamodule. Right now they aren't being used at all.